### PR TITLE
Add toggle for homepage animation, with cookies

### PR DESCRIPTION
**Pull request by juliawu**
Adds a light grey toggle to the bottom-right corner of the animation on the homepage, which when clicked, toggles hiding and showing the animation.

A cookie is set on the browser to save whether the animation was hidden, and if cookie is present, the homepage will start with the animation hidden. The cookie lasts for 24hrs.

https://github.com/datacommonsorg/website/assets/4034366/f3462169-859e-4e88-8a7c-05be398aa593

### CODE REVIEW

File: static/css/homepage.scss [hunk 1]
```diff
@@ -333,6 +333,15 @@ $search-answer-bottom: 48px;
   }
 }
 
+#search-animation-toggle {
+  color:#bbb;
+  font-size: 18px;
+  margin-bottom: $topics-section-vmargin !important;
+  margin-left: auto;
+  margin-top: -24px;
+  width: fit-content;
```
Comment (beets) [inline static/css/homepage.scss:342]: Also add cursor: pointer

**Comment by chejennifer** [comment]
very cool, but can we make: `cursor: pointer` or `cursor: default` for the icon so that it looks clickable

**Review by chejennifer — APPROVED** [review]

**Comment by juliawu** [comment]
> very cool, but can we make: cursor: pointer or cursor: default for the icon so that it looks clickable

Good idea! Done!

**Review by beets — APPROVED** [review]
Thanks for the change!

File: static/css/homepage.scss (hunk 1 above)
Comment (juliawu) [reply static/css/homepage.scss:342]: Done!

**Pull request merged**